src/brpc/server.cpp

```diff
@@ -2197,6 +2197,21 @@ int Server::MaxConcurrencyOf(google::protobuf::Service* service,
     return MaxConcurrencyOf(service->GetDescriptor()->full_name(), method_name);
 }
 
+bool Server::AcceptRequest(Controller* cntl) const {
+    const Interceptor* interceptor = _options.interceptor;
```

Comment:
Most LIKELY, the interceptor is NULL for a non-interception policy. 
So I think we could first check nullity of interceptor and also error_code and error_text will be in smaller scope, for better performance.
```
if (!interceptor) {
  return true;
}

int error_code = 0;
std::string error_text;
if (!interceptor->Accept(cntl, error_code, error_text)) {
        cntl->SetFailed(error_code, "Reject by Interceptor: %s", error_text.c_str());
        return false;
 }

return true;

```